libethereum/Block.h

```diff
@@ -249,6 +243,10 @@ class Block
     /// @returns true if sealed - in this case you can no longer append transactions.
     bool isSealed() const { return !m_currentBytes.empty(); }
 
+    /// Clears the block bytes - typically used in cases where we want to mutate a sealed
+    /// block's state (e.g. eth_call)
+    void unseal() { m_currentBytes.clear(); }
```

Comment:
This shouldn't be needed now when we don't fill `m_currentBytes`? Let's remove it then